plugins/woocommerce-blocks/assets/js/blocks/product-collection/utils.tsx

```diff
@@ -138,35 +137,33 @@ export function getDefaultValueOfFilterable() {
  * This enhancement allows the Core Pagination block to be available for the Product Collection block.
  */
 export const addProductCollectionToQueryPaginationParentOrAncestor = () => {
-	if ( isWpVersion( '6.1', '>=' ) ) {
-		addFilter(
-			'blocks.registerBlockType',
-			'woocommerce/add-product-collection-block-to-parent-array-of-pagination-block',
-			( blockSettings: Block, blockName: string ) => {
-				if ( blockName !== coreQueryPaginationBlockName ) {
-					return blockSettings;
-				}
-
-				if ( blockSettings?.ancestor ) {
-					return {
-						...blockSettings,
-						ancestor: [ ...blockSettings.ancestor, blockJson.name ],
-					};
-				}
+	addFilter(
+		'blocks.registerBlockType',
+		'woocommerce/add-product-collection-block-to-parent-array-of-pagination-block',
+		( blockSettings: Block, blockName: string ) => {
+			if ( blockName !== coreQueryPaginationBlockName ) {
+				return blockSettings;
+			}
 
-				// Below condition is to support WP >=6.4 where Pagination specifies the parent.
-				// Can be removed when minimum WP version is set to 6.5 and higher.
-				if ( blockSettings?.parent ) {
-					return {
-						...blockSettings,
-						parent: [ ...blockSettings.parent, blockJson.name ],
-					};
-				}
+			if ( blockSettings?.ancestor ) {
+				return {
+					...blockSettings,
+					ancestor: [ ...blockSettings.ancestor, blockJson.name ],
+				};
+			}
 
-				return blockSettings;
+			// Below condition is to support WP >=6.4 where Pagination specifies the parent.
+			// Can be removed when minimum WP version is set to 6.5 and higher.
+			if ( blockSettings?.parent ) {
+				return {
+					...blockSettings,
+					parent: [ ...blockSettings.parent, blockJson.name ],
+				};
 			}
```

Comment:
Based on the comment, seems like this could be removed as well. However, can't recall how to test it without digging deeper so to avoid extending this PR scope it's fine to keep it as-is.

Reply:
I prefer not extending the scope of the current PR 😅